packages/cpp/README.md

```diff
@@ -81,14 +81,14 @@ To get this working, you need to enable clangd's global index using the
 
 ## Using the clang-tidy linter
 
-Note: This functionality is available when using clangd 9 and later.
+**Note: This functionality is available when using clangd 9 and later.**
 
-You can set the preference 'cpp.clangTidy' to enable the clang-tidy linter included in clangd. When the preference is set, there are two ways to chose which of its built-in checks clang-tidy will use:
+You can set the preference 'cpp.clangTidy' to enable the clang-tidy linter included in clangd. When the preference is set, there are two ways to choose which of its built-in checks clang-tidy will use:
 
 - using the preferences:  'cpp.clangTidyChecks'
 - using the file '.clang-tidy' . The file is located in the same folder of the files or a parent folder.
```

Comment:
What is meant by the `same folder of the files`?

Reply:
The file ".clang-tidy" can be located in the same folder of the file you try to use the linter.

Would it be better if I put :
OLD:
 using the file '.clang-tidy' . The file is located in the same folder of the files...
NEW
 using the file '.clang-tidy' . This file is located in the same folder of the files...

Reply:
@lmcbout what happens if in a workspace you have multiple `.clang-tidy` files present?
Will it pick up the closest `.clang-tidy` file and apply only those specific rules?

Reply:
> OLD:
> using the file '.clang-tidy' . The file is located in the same folder of the files...
> NEW
> using the file '.clang-tidy' . This file is located in the same folder of the files...

I don't know if my eyes deceive me but these two sentences look the same :see_no_evil:

Reply:
@vince-fugnitto need glasses :)